shopinvader_anonymous_partner/models/res_partner.py

```diff
@@ -101,3 +102,26 @@ def _get_anonymous_partner__cookie(self, cookies: Cookies):
         if not token:
             return self.env["res.partner"].sudo().browse()
         return self._get_anonymous_partner__token(token)
+
+    def _promote_from_anonymous_partner(self, anonymous_partner: ResPartner):
```

Comment:
```suggestion
    def _transfer_from_anonymous_partner(self, anonymous_partner: ResPartner):
```

Transfer sounds better, because promote gives the impression that we transform an existing anonymous partner to a non anonymous one. Here we move data from the anonymous partner to the self of this method.

Reply:
I tend to disagree here, transfer sounds a bit too specific. The aim of this function is to determine what to do when the data describing the user changes but the user don't, be it a cart transfer or an external api call or whatever. It is a promotion in the functional sense, when the same user goes from anonymous to identified, even if in the implementation the anonymous partner is discarded in favor of the existing one.

Reply:
Ok, I don't have a better name in mind. Let's keep it like this.